bgpd/bgp_evpn.c

```diff
@@ -2052,8 +2058,8 @@ static int delete_global_type2_routes(struct bgp *bgp, struct bgpevpn *vpn)
 	safi = SAFI_EVPN;
 
 	rdrn = bgp_node_lookup(bgp->rib[afi][safi], (struct prefix *)&vpn->prd);
-	if (rdrn && rdrn->info) {
-		table = (struct bgp_table *)rdrn->info;
+	if (rdrn && bgp_node_has_info_data(rdrn)) {
+		table = bgp_table_get_node_info(rdrn);
```

Comment:
Slight preference for swapping lines and testing against table -- this is a non-blocking comment.

Reply:
I struggled with this one and went back and forth a couple of times.  I'll switch it.